client/core/core.go

```diff
@@ -1880,6 +1885,9 @@ func (c *Core) authDEX(dc *dexConnection) error {
 
 	dc.compareServerMatches(matches)
 
+	// for _, match := range matches {
+	// 	match.tracker.tick()
+	// }
```

Comment:
Was considering ticking the matches now rather than waiting for the ticker in listen.  Just removing the comment though as i don't want to muddle the responsibilities for doing this.